### ISSUE TITLE
Rule idea: operation-no-unit-adding

### ISSUE THREAD

**Issue by dryoma** [kristerkari/stylelint-scss#68]
Inspired by http://sass-guidelin.es/#units

```scss
$value: 42;

// Good
$length: $value * 1px;

// Bad
$length: $value + px;
```

Again, not sure if it's going to work out.
